.pullapprove.yml

```
@@ -876,6 +877,25 @@ groups:
         - mhevery
 
 
+  # =========================================================
+  #  Docs: Contributors
+  # =========================================================
+  docs-contributors:
+    <<: *defaults
+    conditions:
+      - *can-be-global-approved
+      - *can-be-global-docs-approved
+      - >
+        contains_any_globs(files, [
+          'aio/content/marketing/contributors.json',
+          'aio/content/images/bios/**',
+          ])
+    reviewers:
+      users:
+        - twerske
+        - mgechev
```

We generally try to keep users alphabetically ordered :pray:

```suggestion
        - mgechev
        - twerske
```

---

Thanks for the feedback @gkalpak 👍 I've updated the ordering.